Remnant culture mission on sign fonts (#6738)
endless-sky/endless-sky: data/remnant/remnant 1 introduction.txt

			`	She looks at you quizzically for a moment, and then her display shifts to a mass of symbols. She skims briefly. "I have never even heard of such a unit. Apparently it is a very old one, though. Our records say that it had largely stopped being used by early in the 21st century. Well, I learned something new today." The display and its field of symbols vanishes as she slings her backpack over her shoulder.`
			label stormend
			`	"Thanks for the ride, Captain. I have to get working on this data, but if you are interested in helping some more, I hope to have more research expeditions planned soon."`

mission "Remnant: Sign Fonts"
	invisible
	minor
	source "Viminal"
	to offer
		has "Remnant: Learn Sign Follow Up: offered"
	on offer
		conversation

> 			`You almost never enter a Remnant outfitter from the public entrance, the way you would in human space. You come through the repair bays dockside, past the technicians working on an ibis with an asterioid scanner and a couple of inhibitor cannons stacked next to it. You are walking along side a hover cart with a triple plasma core and some heat shunts that are destined for the refurbishing team. Once receiving has paid you for the salvage, you head past the small break room with the worn seating but very good coffee machine and into the small showroom.`

Reviewer: ```suggestion
			`You almost never enter a Remnant outfitter from the public entrance, the way you would in human space. You come through the repair bays dockside, past the technicians working on an Ibis with an asteroid scanner and a couple of Inhibitor Cannons stacked next to it. You are walking along side a hover cart with a Korath plasma core and some heat shunts that are destined for the refurbishing team. Once receiving has paid you for the salvage, you head past the small break room with the worn seating but very good coffee machine and into the small showroom.`
```
Paid me for the salvage? Why would I rarely enter through the public entrance for Remnant outfitters alone?

			`	There are a few open workstations with terminals and holo displays that allow you to review specifications and technical manuals on all the systems for sale, as well as a fit checker that allows you to pull up your ship and verify that it will all fit. Once you are happy with the order, you specify your landing pad and the crew gets to work. You've just pulled up the <ship> on the display when Aeolus walks up.`

> 			`	"I have beeen watching you sign with the crews," he says, "and I thought of something that might help you. May I?"`

Reviewer: ```suggestion
			`	"I have been watching you sign with the crews," he says, "and I thought of something that might help you. May I?"`
```

			`	You move over and let him have the terminal. He moves through several layers of the software you didn't know existed until he finds a long list of communication options. He selects one that looks like two hands and places the order.`
			`	"This is a display font," he explains. "It will display flight control and hail messages to your ship in the same way as the spaceport displays do. You can also sync it down to your commlink if you wish."`

> 			`	He pulls out his own commlink and shows you a notification in the stylised lines marking arm and hand positions that you've come to recognize from exit signs, toilets, departure notices, and civil defense messages.`

Reviewer: ```suggestion
			`	He pulls out his own commlink and shows you a notification in the stylized lines marking arm and hand positions that you've come to recognize from exit signs, toilets, departure notices, and civil defense messages.`
```
